prowler/providers/aws/aws_provider.py

```diff
@@ -84,8 +90,9 @@ def __init__(
         scan_unused_services: bool = None,
         resource_tags: list[str] = [],
         resource_arn: list[str] = [],
-        audit_config: dict = {},
+        config_file: str = None,
```

Comment:
Why are you changing this? We should support passing the config as a `dict`.

Reply:
True, good point